### ISSUE TITLE
Failure to run PkgDev.publish()

### ISSUE THREAD

**Issue by richardreeve** [JuliaLang/METADATA.jl#8740]
I'm sure I'm doing something very stupid, but I can't work it out. I can't publish a new tag to `METADATA` - the `PkgDev.tag()` itself worked fine, but I get some fatal error every time I try to publish it:

```
julia> PkgDev.publish()
INFO: Validating METADATA
ERROR: packages with unsatisfiable requirements found:
    BioEnergeticFoodWebs v0.3.0 – no valid versions exist for package DiffEqProblemLibrary
    BioEnergeticFoodWebs v0.3.1 – no valid versions exist for package DiffEqProblemLibrary
```

This happens even though I deleted the `.julia/v0.5` folder and reinstalled all of the packages (with or without `BioEnergeticFoodWebs` added)... any suggestions? I know it's **much** more likely to be something to do with me, but is it a problem with `PkgDev` or `METADATA`? I've put this in PkgDev too just in case it's more relevant there... sorry. JuliaLang/PkgDev.jl#106

**Comment by tkelman** [comment]
What Julia version are you using?

**Comment by richardreeve** [comment]
0.5.1... when I try it with the 0.6 pre beta it doesn't manage to do the tag before failing...

**Comment by tkelman** [comment]
Ah I'm seeing this too now. It looks like I need to backport more resolver changes

**Comment by richardreeve** [comment]
Phew. I'm not going mad!

**Comment by dpsanders** [comment]
Have you tried using Attobot?

**Comment by richardreeve** [comment]
Hmm, I just set up attobot and tagged the repository on github with the same tag, and nothing happened. I've obviously never used it before, but there's no PR on METADATA which seems to be what's supposed to happen...

**Comment by richardreeve** [comment]
Actually, Attobot does seem to work now I'm using it correctly.

**Comment by tkelman** [comment]
this should be fixed in 0.5.2
